src/capi_frontend/capi.cpp

```diff
@@ -936,6 +1081,36 @@ DLL_PUBLIC OVMS_Status* OVMS_GetServableState(OVMS_Server* serverPtr, const char
     *state = modelInstance->getStatus().isFailedLoading() ? OVMS_STATE_LOADING_FAILED : static_cast<OVMS_ServableState>(static_cast<int>(modelInstance->getStatus().getState()) / 10 - 1);
     return nullptr;
 }
+DLL_PUBLIC OVMS_Status* OVMS_GetServableContext(OVMS_Server* serverPtr, const char* servableName, int64_t servableVersion, void** oclContext) {
```

Comment:
```suggestion
DLL_PUBLIC OVMS_Status* OVMS_GetServableOclContext(OVMS_Server* serverPtr, const char* servableName, int64_t servableVersion, void** oclContext) {
```
Add "Ocl" in case we will have other contexts in future?

Reply:
@atobiszei